test/distributed/tensor/test_utils.py

```diff
@@ -19,6 +22,80 @@
 c10d_functional = torch.ops.c10d_functional
 
 
+class LocalTest(TestCase):
+    def test_explicit_order_placements(self):
+        # mesh_shape: ShapeType, placements: Sequence[Placement]
+        test_cases = [
+            {
+                "mesh_shape": [2, 2],
+                "placements": [Replicate()],
+                "ordered": [(0, Replicate())],
+            },
+            {
+                "mesh_shape": [2, 2],
+                "placements": [Shard(0)],
+                "ordered": [(0, Shard(0))],
+            },
+            {
+                "mesh_shape": [2, 2],
+                "placements": [_StridedShard(0, split_factor=2), Shard(0)],
+                "ordered": [(1, Shard(0)), (0, Shard(0))],
+            },
+            {
+                "mesh_shape": [2, 3, 4],
+                "placements": [Shard(0), _StridedShard(0, split_factor=4), Shard(0)],
```

Comment:
should add a test have SS in the leftmost of the placements

Reply:
oh, i actually already do.  I have the one right above this one with (SS, S), and below I have the error case for NYI for (SS, SS, S).  I think i can just finish that implementation now though.